src/js/_enqueues/admin/site-icon.js

```diff
@@ -1,138 +1,180 @@
-(function($) {
+/* global wp, jQuery */
```

Comment:
I previously reported that to add the comment for globals because I seem to recall the lack of uaed to make jshint complain. I may be wrong though, if that's the case I stand corrected. These gloabls are already registered in thte `.jshintrc` file. See https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#globals

Reply:
I would keep these even if no longer necessary for `jshint` since every neighbor file has them. Additionally, it does make it easier to understand what the file relies on.

Reply:
The comment on globals is now back in top of the file.